tracpro/settings/base.py

```diff
@@ -333,7 +333,7 @@ def _org_scheduler_task(task_name):
         'delete',  # can delete an object
         'list',  # can view a list of the objects
     ),
-    'orgs.org': ('create', 'update', 'list', 'edit', 'home'),
+    'orgs.org': ('create', 'update', 'list', 'edit', 'home', 'fetch_runs'),
```

Comment:
@rowanseymour yes, it is there:
`Group.objects.get(name='Administrators').permissions.filter(content_type=content_type)`
`[<Permission: orgs | org | Can edit org>, <Permission: orgs | org | Can fetch_runs org>, <Permission: orgs | org | Can home org>]`